### PR TITLE
Skill objects were breaking things

### PR DESCRIPTION
**Pull request by lkduerig**

### CODE REVIEW

File: public/js/filters/filters.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ angular.module('rmtFilters', [])
 
 .filter('sanitizeSkill', function () {
     return function (text) {
+      if (!_.isString(text))
```
Comment (amolnw2778) [inline public/js/filters/filters.js]: Better - use braces for `if` blocks.

File: public/routes.js [hunk 2]
```diff
@@ -84,10 +81,9 @@ module.exports = function (router) {
       Person.remove({
         _id: req.params.person_id
       }, function (err, person) {
```
Comment (amolnw2778) [inline public/routes.js]: `person` param is unused and can be removed.

File: public/routes.js [hunk 3]
```diff
@@ -165,8 +164,9 @@ module.exports = function (router) {
       Project.remove({
         _id: req.params.project_id
       }, function (err, project) {
```
Comment (amolnw2778) [inline public/routes.js]: `project` param is unused and can be removed.

File: public/routes.js [hunk 4]
```diff
@@ -14,24 +14,23 @@ module.exports = function (router) {
       person.role = req.body.role;
```
Comment (amolnw2778) [inline public/routes.js]: This can be made better I think.
`person = {
  role: req.body.role,
  email: req.body.email
  ...
}`

Comment (lkduerig) [inline public/routes.js]: It does not seem so. Creating a new Person() on previous line initializes the object with an _id parameter that is required.

Comment (amolnw2778) [inline public/routes.js]: oh.. ok

**Pull request merged**